lib/tasks/database_updates/backfill_publisher_selected_wallet.rake

```diff
@@ -0,0 +1,105 @@
+namespace :database_updates do
+  desc 'Backfill Publisher Selected Wallets'
+  task :backfill_publisher_selected_wallets => :environment do
+    def handle_gemini
+      now = Time.zone.now
+      gemini_to_set = Publisher.
+        includes([:gemini_connection]).
+        joins(:gemini_connection).
```

Comment:
Would you use `joins` in this case? I don't think there's any performance gain by using `includes` in our case

Reply:
Joins doesn't automatically eager load the data, which I need when getting the connection id. 

Well, this used to be relevant until I switched to pluck -- since it's array based now, then includes is unnecessary. I can remove it tomorrow.

Reply:
Done.